depatment_project/department_app/models/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+
+class Department(models.Model):
+    id = models.AutoField(primary_key=True, editable=False)
+    Department_name = models.CharField(max_length=30, null=False, unique=True)
```

Comment:
Why you start the attribute name with the capital letter? This is not a pythonic approach.
It makes sense to call this attribute name instead of Department name.

department.name looks better than department.department_name